We handle stripping of the binary through dh_strip, thus remove the
upstream code handling that, which does not care about our Debian
build options.

Index: aiccu-20070115/unix-console/Makefile
===================================================================
--- aiccu-20070115.orig/unix-console/Makefile	2009-10-25 22:31:57.000000000 +0100
+++ aiccu-20070115/unix-console/Makefile	2009-10-25 22:32:04.000000000 +0100
@@ -145,11 +145,6 @@
 
 aiccu:	$(OBJS) ${SRCS} ${INCS}
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(OBJS)
-ifeq ($(shell echo $(CFLAGS) | grep -c "DEBUG"),0)
-ifeq ($(shell echo "$(RPM_OPT_FLAGS)" | wc -c),1)
-	strip $@
-endif
-endif
 
 clean:
 	$(RM) -f $(OBJS) aiccu
